core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -145,6 +145,21 @@ sealed abstract class Chunk[+A] extends ChunkLike[A] with Serializable { self =>
     Chunk.BitChunkByte(Chunk.fromArray(arr), 0, bits)
   }
 
+  /**
+   * Crates a base64 encoded string based on the chunk's data.
+   */
+  def asBase64String(implicit ev: Chunk.IsText[A]): String = {
```

Comment:
base64 is a binary-to-text encoding, would this make more sense:
```suggestion
  def asBase64String(implicit ev: A <:< Byte): String = {
```